package.json

```diff
@@ -105,7 +105,7 @@
 		"@hashicorp/react-button": "^6.2.1",
 		"@hashicorp/react-call-to-action": "^4.1.1",
 		"@hashicorp/react-checkbox-input": "^5.0.2",
-		"@hashicorp/react-code-block": "^6.2.0",
+		"@hashicorp/react-code-block": "^6.2.1-canary-20221019235743",
```

Comment:
🚧 Will bump to stable once https://github.com/hashicorp/react-components/pull/768 lands 👍

Reply:
✅ released & bumped